src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheckTest.java

```diff
@@ -321,6 +324,102 @@ public void testWrongSingletonTagInJavadoc() throws Exception {
                 expected);
     }
 
+    @Test
+    public void testNonTightHtmlTagIntolerantCheck() throws Exception {
+        final DefaultConfiguration checkConfig =
+                createModuleConfig(NonTightHtmlTagIntolerantCheck.class);
+        checkConfig.addAttribute("violateExecutionOnNonTightHtml", "true");
+        final String[] expected = {
+            "6: " + getCheckMessage(MSG_UNCLOSED_HTML, "p"),
+            "13: " + getCheckMessage(MSG_UNCLOSED_HTML, "p"),
+            "16: " + getCheckMessage(MSG_UNCLOSED_HTML, "li"),
+            "21: " + getCheckMessage(MSG_UNCLOSED_HTML, "p"),
+            "27: " + getCheckMessage(MSG_UNCLOSED_HTML, "tr"),
+            "34: " + getCheckMessage(MSG_UNCLOSED_HTML, "li"),
+            "54: " + getCheckMessage(MSG_UNCLOSED_HTML, "li"),
+            "71: " + getCheckMessage(MSG_UNCLOSED_HTML, "p"),
+            "80: " + getCheckMessage(MSG_UNCLOSED_HTML, "tr"),
+            "124: " + getCheckMessage(MSG_UNCLOSED_HTML, "p"),
+        };
+        verify(checkConfig, getPath("InputAbstractJavadocNonTightHtmlTags.java"), expected);
+    }
+
+    @Test
+    public void testNonTightHtmlTagIntolerantCheckReportingNoViolation() throws Exception {
+        final DefaultConfiguration checkConfig =
+                createModuleConfig(NonTightHtmlTagIntolerantCheck.class);
+        final String[] expected = CommonUtils.EMPTY_STRING_ARRAY;
+        verify(checkConfig, getPath("InputAbstractJavadocNonTightHtmlTags.java"), expected);
+    }
+
+    @Test
+    public void testNonTightHtmlTagIntolerantCheckVisitCount()
+            throws Exception {
+        final DefaultConfiguration checkConfig =
+                createModuleConfig(NonTightHtmlTagIntolerantCheck.class);
+        checkConfig.addAttribute("reportVisitJavadocToken", "true");
```

Comment:
Please add also test with `checkConfig.addAttribute("reportVisitJavadocToken", "true");` and `checkConfig.addAttribute("violateExecutionOnNonTightHtml", "true");`

Reply:
Done.